include/git2/attr.h

```diff
@@ -238,8 +238,11 @@ GIT_EXTERN(int) git_attr_foreach(
  * disk no longer match the cached contents of memory.  This will cause
  * the attributes files to be reloaded the next time that an attribute
  * access function is called.
+ *
+ * @param repo The repository containing the gitattributes cache
+ * @return 0 on success, non-zero callback return value, or error code
```

Comment:
Copy & Paste error from the git_attr_foreach above.
It should read
```suggestion
 * @return 0 on success or an error code on failure
```

Reply:
Oops!  Thanks.  :)